templates/404.html

```
@@ -0,0 +1,42 @@
+{% extends "base.html" %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)